### PR TITLE
create eagerQueryAsync, execQueryAsync -> return a Promise

### PR DESCRIPTION
**Pull request by tttmaximttt**

### CODE REVIEW

File: lib/Drivers/DML/_shared.js [hunk 1]
```diff
@@ -10,7 +28,18 @@ module.exports = {
     }
     return this.execSimpleQuery(query, cb);
   },
-  eagerQuery: function (association, opts, keys, cb) {
+
+  execQueryAsync: function () {
+    var args = arguments;
+    if (args.length === 1) {
```
Comment (anton-locomote) [inline lib/Drivers/DML/_shared.js]: What if we have another variant of args.length, example 3?
As a variant we could use 
`var args = arguments;`
 `var query = '<something>';`
 `if (args.length === 1) {`
      `query = args[0];`
 `} else if (args.length === 2) {`
      `query = this.query.escape(args[0], args[1]);`
 `}`
`return this.execSimpleQueryAsync(query); // so we could always return something here, but correct me if I'm wrong :)`

File: lib/Drivers/DML/sqlite.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ var sqlite3 = require("sqlite3");
 var Query   = require("sql-query").Query;
 var shared  = require("./_shared");
 var DDL     = require("../DDL/SQL");
+var Promise     = require("bluebird");
```
Comment (anton-locomote) [inline lib/Drivers/DML/sqlite.js]: `var Promise <needs less spaces here> = require("bluebird");`

**Comment by tttmaximttt** [comment]
@dxg Sounds strange, because we discussed it when we talked about this, and we decided that it was better to do this manually. Really I don't remember your reasons or maybe I didn't understand you right.

**Comment by dxg** [comment]
The idea is we'll have less duplicate code if we `Promise.promisify` vs reimplementing. That's the main difference.

**Pull request merged**